### PR TITLE
Kriston issue#139 update admin housing program page

### LINKED ISSUE #139: Update Admin Housing Program Page
When logged in as admin, after editing an entry for the housing program on ```/About/HousingProgramData``` and clicking "Update Entry", the page refreshes and the client starts on the top of the page. Updates could be done via JavaScript or using an on page anchor link, start the client on the table. 
This is low priority as it is a minor inconvenience and updates typically happen once per year

### PR DESCRIPTION
**Pull request by dauthi66**
closes #139 

This should dynamically create anchor tags based on household size, which should be a unique number in this case. I'm not sure how to test if it is pulling to the correct entry.

### CODE REVIEW

File: PC2/Views/About/HousingProgramData.cshtml [hunk 1]
```diff
@@ -39,7 +39,9 @@
                 <form method="post">
                     <input type="hidden" name="HouseHoldSize" value="@item.HouseHoldSize">
                     $<input name="MaximumIncome" value="@item.MaximumIncome" />
-                    <input type="submit" value="Update Entry" class="btn btn-primary">
+                    <a href="#@item.HouseHoldSize">
```
Comment (JoeProgrammer88) [inline PC2/Views/About/HousingProgramData.cshtml:42]: This link gets lost since the controller returns a brand new view

**Review by JoeProgrammer88 — CHANGES_REQUESTED** [review]
To test something like this you can shrink your browser window so it should scroll down the page a bit. Your solution is not working for me. The problem since this posts to the server and then returns a brand new page, is your link to the anchor tag is lost. For the anchor tags we also need elements with that specific id to jump to. For example:
```html
<a href="#1">Link text</a>
<!-- Would jump to an element with that id -->
<p id="1"></p>
```
Two potential solutions. Use JS and AJAX to post an update to the server so the page does not get refreshed. Or, if the if(TempData["Message"] check is true, you can add some JS in there to scroll to the HousingProgram table by using an id or another method. We don't necessarily need to jump to the specific entry that was edited since the table will remain with a small amount of entries

File: PC2/Views/About/HousingProgramData.cshtml (hunk 1 above)
Comment (dauthi66) [reply PC2/Views/About/HousingProgramData.cshtml]: That's what I was afraid of. I'm not sure why it was sometimes working on my end.

**Review by JoeProgrammer88 — APPROVED** [review]
Your changes look great! Since the id attribute value is not being used we can take the id attribute off of the submit button

**Review by JoeProgrammer88 — APPROVED** [review]
Looks great!

**Pull request merged**